### ISSUE TITLE
Error in the labelled .nii volume

### ISSUE THREAD

**Issue by ftadel** [simnibs/simnibs#117]
Running the following command:
```
charm --forceqform FEM_SimNIBS4 FEM_SimNIBS4T1.nii FEM_SimNIBS4T2.nii
```
With the following files:
[subject01_charm.zip](https://github.com/simnibs/simnibs/files/10757896/subject01_charm.zip)

Produces a file `final_tissues.nii.gz` with a wrong field of view:
![image](https://user-images.githubusercontent.com/6920058/219421141-da62437e-1c42-4438-9b7e-8e34361211c0.png)

**Comment by oulap** [comment]
Hi,

This was a header problem related to issue #110  that haven't gotten around to fix yet.
I'm attaching a T1 with a fixed header here, try running with that.
[T1_fixed.nii.gz](https://github.com/simnibs/simnibs/files/10808931/T1_fixed.nii.gz)

**Comment by ftadel** [comment]
I reached the same conclusion yesterday.
I noticed that rewriting the volumes with a QFORM and no SFORM before calling SimNIBS worked.

Btw, this is for the integration of SimNIBS 4 within Brainstorm:
https://neuroimage.usc.edu/brainstorm/Tutorials/FemMesh#SimNIBS

**Comment by oulap** [comment]
Aah, great! Let me know if you need help with that. Just noticed that it says this on the website:

" It also creates high-resolution cortex surfaces, but as it does not include CAT12 anymore, it does not provide surface-based parcellations. For surface and volume-based atlases, it might be necessary to execute CAT12 on top of SimNIBS."

We do still run the CAT12 binaries so surface parcellations and registration to and from fsaverage work the same way as in the previous versions of SimNIBS.

**Comment by ftadel** [comment]
> We do still run the CAT12 binaries so surface parcellations and registration to and from fsaverage work the same way as in the previous versions of SimNIBS.

Oh I see, thanks for pointing at this, I did not realized the surface registration was still done with CAT12.
From the CAT12 outputs, we were importing all the surface and volume parcellations, so this is missing compared with headreco.
I replaced the text with:

_It also creates high-resolution cortex surfaces registered with the FSAverage template, but it does not generate the full CAT12 output folder anymore. For surface and volume-based atlases, it might be necessary to execute CAT12 from Brainstorm on top of SimNIBS. _
